src/.jshintrc

```diff
@@ -1,22 +0,0 @@
-{
```

Comment:
There's no need for the `src` directory to extend the root JSHint configuration anymore; it can simply use the base rules.